Formula/netlify-cli.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class NetlifyCli < Formula
+  desc "Netlify command-line tool"
+  homepage "https://www.netlify.com/docs/cli"
+  url "https://registry.npmjs.org/netlify-cli/-/netlify-cli-2.11.23.tgz"
```

Comment:
@cglong any way to make this dynamic?

Reply:
Unfortunately no; Homebrew requires that [formulae be versioned](https://docs.brew.sh/Acceptable-Formulae#we-dont-like-install-scripts-that-download-unversioned-things) and reproducible. This shouldn't be a problem practically though, as PRs to bump the versions typically get opened within a day or two 🙂